### DISCUSSION TITLE
ccf-validation-tools/Kidney/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#227] [General]
# ccf-validation-tools/Kidney/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Kidney/comments/

**Comment by emquardokus** [comment]
Kidney_v1.3_DRAFT https://docs.google.com/spreadsheets/d/1aXnWkwArSRmDnhyq0n_ptISWnmRwXvD4X0BPa9WQfmg/edit#gid=949267305
Different labels
1) Fixed In row 46, the term CL:4030021 has different name/label in the source ontology. The name/label in the ASCT+B table is connecting tubule intercalated cell type B and the one in the ontology is kidney connecting tubule beta-intercalated cell. 
2. Fixed uberon ID to correct one juxtaglomerular arteriole	UBERON:0034884 not UBERON:0001916
In row 60, the term UBERON:0001916 has different name/label in the source ontology. The name/label in the ASCT+B table is juxtaglomerular arteriole and the one in the ontology is endothelium of arteriole
3. Fixed In row 45, the term CL:4030020 has different name/label in the source ontology. The name/label in the ASCT+B table is connecting tubule intercalated cell type A and the one in the ontology is kidney connecting tubule alpha-intercalated cell.
4. Fixed; In row 44, the term CL:4030019 has different name/label in the source ontology. The name/label in the ASCT+B table is connecting tubule intercalated cell and the one in the ontology is kidney connecting tubule intercalated cell. 
5. Fixed: In row 70, the term CL:4030022 has different name/label in the source ontology. The name/label in the ASCT+B table is medullary fibroblast and the one in the ontology is renal medullary fibroblast. 
6. fixed; In row 20, the term UBERON:0001284 has different name/label in the source ontology. The name/label in the ASCT+B table is renal column (column of Bertin) and the one in the ontology is renal column.
7. fixed typo: In row 59, the term CL:0000115 has different name/label in the source ontology. The name/label in the ASCT+B table is endothlial cell and the one in the ontology is endothelial cell. For reference, the given name/label by SMEs is Endothelial Cell.
8. fixed: In row 54, the term CL:4030015 has different name/label in the source ontology. The name/label in the ASCT+B table is collecting duct intercalated cell type A and the one in the ontology is kidney collecting duct alpha-intercalated cell. 
9. fixed: In row 43, the term CL:4030018 has different name/label in the source ontology. The name/label in the ASCT+B table is connecting tubule principal cell and the one in the ontology is kidney connecting tubule principal cell.
10 fixed rows:42-46  In row 42, the term UBERON:0005097 has different name/label in the source ontology. The name/label in the ASCT+B table is connecting tubule and the one in the ontology is renal connecting tubule.
11.Replaced obsolete with merged term ID UBERON:0005751 in rows 13, 14, 20 and 23 In row 13, the term UBERON:0006852 has different name/label in the source ontology. The name/label in the ASCT+B table is glomerular visceral epithelium and the one in the ontology is obsolete glomerular visceral epithelium.
12. fixed 9 instances rows: 17, 30-34
In row 17, the term UBERON:0001289 has different name/label in the source ontology. The name/label in the ASCT+B table is descending thin limb of loop of Henle and the one in the ontology is descending limb of loop of Henle. 
13, fixed In row 34, the term UBERON:0004193 has different name/label in the source ontology. The name/label in the ASCT+B table is ascending thin limb of loop of Henle and the one in the ontology is loop of Henle ascending limb thin segment.

**Comment by emquardokus** [comment]
Kidney_v1.3_DRAFT https://docs.google.com/spreadsheets/d/1aXnWkwArSRmDnhyq0n_ptISWnmRwXvD4X0BPa9WQfmg/edit#gid=949267305
**Blank ontology ID**
Several of these were added from New term request google sheet: https://docs.google.com/spreadsheets/d/1BihrOU9vWrsA_wYjcbdPUr42_Sb22CO3gkBDieiYQPs/edit#gid=1063693910

**This report seems old as some of the blank ontology IDs were added to CL as new term requests quite a while ago and the table was updated in the last v1.2 release.**
These problems were resolved in the previous release table:
1. In row 17, no term id was found for the name/label Descending Thin Limb Cell Type 2. is as follows: 
Descending Thin Limb Cell Type 2	kidney loop of Henle long descending thin limb outer medulla epithelial cell	CL:4030013

**Fixed in draft v1.3: **
1. .Fixed;  In row 63, no term id was found for the name/label Endothelium (non glomerular).
2. Fixed; In row 64, no term id was found for the name/label Endothelium (non glomerular).
3. Removed "immune" from  AS/3 column and added it instead to AS/2/NOTES column; In rows 72 through 88, no term id was found for the name/label Immune.
4. Fixed  incorrect labe used; meant regulatory T cell, added CL:0000815 In row 84, no term id was found for the name/label Cytotoxic T Cell.

**Things that remain: Probable new term requests; asked author re: definitions and references**
1. In row 12, no term id was found for the name/label capsule mesenchymal stromal cell.
2. In row 17, no term id was found for the name/label descending thin limb of loop of Henle 2.
3. In row 31, no term id was found for the name/label descending thin limb of loop of Henle 1.
4. In row 32, no term id was found for the name/label descending thin limb of loop of Henle 2.
5. In row 33, no term id was found for the name/label descending thin limb of loop of Henle 3.
6. In row 52, no term id was found for the name/label Transitional principal-intercalated cell.
7. In row 89, no term id was found for the name/label Papillary Tip Epithelium

**Comment by emquardokus** [comment]
**AS-AS relationship corrections**

Row 24 renal glomerulus	renal glomerulus	UBERON:0000074 replaced generic blood vessel and resolves 

Removed  9 instances of UBERON:0001981 	blood vessel 	Blood Vessel 	 where it was was parent of UBERON:0006544 	kidney vasculature ; added notes column for author to convey blood vasculature and lymph vasculature in table.

Removed instance of UBERON:0004190 	renal glomerulus vasculature 	Glomerulus Vasculature 	UBERON:0001981 	blood vessel and replaced with renal glomerulus--> renal glomerulus vasculature which has part of relationship in uberon.

renal glomerulus	renal glomerulus	UBERON:0000074--> Glomerulus Vasculature	renal glomerulus vasculature	UBERON:0004190--> glomerular capillary endothelium	glomerular capillary endothelium	UBERON:0004294


**outstanding issues that need ontology support**
1) No lymphatic vasculature specific to kidney (renal), there are instances to support other organs like colon, respiratory system.
2) Juxaglomerular apparatus --discussing with author about ontology relationships vs depiction in table.

Removed instnaces of UBERON:0002303 	juxtaglomerular apparatus 	Juxtaglomerular Apparatus 	UBERON:0001981 	blood vessel to resolve part of relationship

**Comment by emquardokus** [comment]
@aleixpuigb **New term requests kidney tab: https://docs.google.com/spreadsheets/d/1BihrOU9vWrsA_wYjcbdPUr42_Sb22CO3gkBDieiYQPs/edit#gid=1063693910**

**Reply by emquardokus** [reply]
Filed uberon ticket June 7, 2023: Revise subclass relationship: vasa recta #2914 
https://github.com/obophenotype/uberon/issues/2914

**Reply by aleixpuigb** [reply]
Thank you, @emquardokus

**Comment by emquardokus** [comment]
Kidney_v1.4-DRAFT 
https://docs.google.com/spreadsheets/d/19B_iDwpVTzLl6JLUl7g943p8b14YxYRwOshm-PLbIwk/edit#gid=949267305

Edits made 9-13-2023 after validation run; changes will be reflected in 9-20-2023 reports Ellen

**Blank ontology ID**


**Added stromal cell parent as CT/1 stromal cell	stromal cell	CL:0000499; however, new term requested**
In row 12, no term id was found for the name/label capsule mesenchymal stromal cell.
@aleixpuigb what is status of new term requests for kidney: https://docs.google.com/spreadsheets/d/1BihrOU9vWrsA_wYjcbdPUr42_Sb22CO3gkBDieiYQPs/edit#gid=1063693910
**New term requests:**
In row 17, no term id was found for the name/label descending thin limb of loop of Henle 2.
In row 31, no term id was found for the name/label descending thin limb of loop of Henle 1.
In row 32, no term id was found for the name/label descending thin limb of loop of Henle 2.
In row 33, no term id was found for the name/label descending thin limb of loop of Henle 3.
In row 65, no term id was found for the name/label Kidney Lymphatic Vessel.
In row 89, no term id was found for the name/label Papillary Tip Epithelium.

Added kidney collecting duct epithelial cell CL:1000454 as CT/1; however, New term requested: In row 52, no term id was found for the name/label Transitional principal-intercalated cell.

Fixed added UBERON:0002113  In row 13, no term id was found for the name/label kidney.

**the lineage of this cell type is difficult and being discussed.** In row 81, no term id was found for the name/label Monocyte Derived Cell; however, a new term is requested;

**Comment by emquardokus** [comment]
Kidney_v1.4-DRAFT
https://docs.google.com/spreadsheets/d/19B_iDwpVTzLl6JLUl7g943p8b14YxYRwOshm-PLbIwk/edit#gid=949267305

**Blank ontology ID missing parent**
ADDED mononuclear phagocyte	CL:0000113 as parent CT/1 to New term request In row 81, the term Monocyte Derived Cell without ontology ID has no parent that is from the CL ontology.

**Comment by emquardokus** [comment]
Kidney v1.4 finalizaiton metadata validation pipeline updates:
https://docs.google.com/spreadsheets/d/19B_iDwpVTzLl6JLUl7g943p8b14YxYRwOshm-PLbIwk/edit#gid=949267305

**Validation feedback for Missing Uberon or CL IDs?**
WARNING: Missing Uberon/CL ID at Column: AC, Row: 12 (Code 6)	NTR capsule mesenchymal stromal cell
WARNING: Missing Uberon/CL ID at Column: AP, Row: 15 (Code 6)	"fixed: FXYD4	FXYD domain containing ion transport regulator 4	HGNC:4028"
WARNING: Missing Uberon/CL ID at Column: Q, Row: 17 (Code 6)	NTR: descending thin limb of loop of Henle 2
WARNING: Missing Uberon/CL ID at Column: AX, Row: 30 (Code 6)	"fixed: SLC44A5	solute carrier family 44 member 5	HGNC:28524 "
WARNING: Missing Uberon/CL ID at Column: Q, Row: 31 (Code 6)	NTR: descending thin limb of loop of Henle 1
WARNING: Missing RDFS Label for ID HGNC:2303 at Column: BE, Row: 31 (Code 6)	"fixed: CPE	carboxypeptidase E	HGNC:2303"
WARNING: Missing Uberon/CL ID at Column: Q, Row: 32 (Code 6)	NTR: descending thin limb of loop of Henle 2
WARNING: Missing Uberon/CL ID at Column: Q, Row: 33 (Code 6)	NTR: descending thin limb of loop of Henle 3
WARNING: Missing RDFS Label for ID HGNC:2026 at Column: BA, Row: 34 (Code 6)	fixed chloride voltage-gated channel Ka
WARNING: Missing Uberon/CL ID at Column: AP, Row: 46 (Code 6)	"fixed: TLDC2	TBC/LysM-associated domain containing 2	HGNC:16112"
WARNING: Missing Uberon/CL ID at Column: AC, Row: 52 (Code 6)	"NTR: Transitional principal-intercalated cell			tPC-IC"
WARNING: Missing Uberon/CL ID at Column: J, Row: 65 (Code 6)	NTR: Kidney Lymphatic Vessel
WARNING: Missing Uberon/CL ID at Column: AC, Row: 81 (Code 6)	"NTR: Monocyte Derived Cell			MDC"
WARNING: Missing Uberon/CL ID at Column: AH, Row: 81 (Code 6)	"fixed: MSR1	macrophage scavenger receptor 1	HGNC:7376"
WARNING: Missing Uberon/CL ID at Column: AL, Row: 81 (Code 6)	"fixed: ITGAX	fms related receptor tyrosine kinase 3	HGNC:6152"
WARNING: Missing Uberon/CL ID at Column: AP, Row: 81 (Code 6)	"fixed: HLA-DQA1	major histocompatibility complex, class II, DQ alpha 1	HGNC:4942"
WARNING: Missing Uberon/CL ID at Column: Q, Row: 89 (Code 6)	NTR: Papillary Tip Epithelium
WARNING: Missing Uberon/CL ID at Column: AP, Row: 89 (Code 6)	"fixed: GPX2	glutathione peroxidase 2	HGNC:4554"

**Comment by aleixpuigb** [comment]
**New terms from other projects (BICAN, CAP)**

CL:4030066 [ureteric bud cell](http://purl.obolibrary.org/obo/CL_4030066)

**Comment by aleixpuigb** [comment]
New terms added in Uberon in the last release (v2024-01-18):

Kidney: 'kidney loop of Henle long descending thin limb outer medulla' (UBERON:8600036)
Kidney: 'kidney loop of Henle long descending thin limb inner medulla' (UBERON:8600037)

**Comment by emquardokus** [comment]
**Kidney v1.5 https://docs.google.com/spreadsheets/d/1SqNmcPDB8PrZF1BhzgdKBxkfLcCR8VYMAkSIbnK_AXA/edit#gid=949267305**


**Blank ontology ID**
1. No term id was found for the name/label descending thin limb of loop of Henle 2 in the following 2 rows 17, 32.
ADD descending thin limb of loop of Henle 2	long descending limb of the loop of Henle in the outer medulla	UBERON:8600036
2. No term id was found for the name/label descending thin limb of loop of Henle 1 in the following 1 row 31.
**From MIRACL for Kidney, https://docs.google.com/spreadsheets/d/1BihrOU9vWrsA_wYjcbdPUr42_Sb22CO3gkBDieiYQPs/edit#gid=1063693910**
Use descending thin limb of loop of Henle 1	short descending thin limb	UBERON:0005099
3. No term id was found for the name/label descending thin limb of loop of Henle 3 in the following 1 row 33.
ADD descending thin limb of loop of Henle 3	long descending limb of the loop of Henle in the inner medulla	UBERON:8600037

**Comment by emquardokus** [comment]
**Kidney_v1.6_DRAFT**
https://docs.google.com/spreadsheets/d/1CS-Tplkosorg8q6Ci8oALTvt9ufvEyDmCNAI2LsKM_g/edit?gid=949267305#gid=949267305

**Different labels**

1. Updated 3 instances to descending vasa recta; recent change in uberon. The term UBERON:0009202 has a different name/label in the source ontology in the following 6 rows 18, 18, 21, 21, 63, 63. The name/label in the ASCT+B table is vasa recta descending limb and the one in the ontology is descending vasa recta. For reference, the given name/label by SMEs is Descending Vasa Recta. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2. updated 2 instances to kidney inner medulla collecting duct principal cell The term CL:1000718 has a different name/label in the source ontology in the following 1 row 51. The name/label in the ASCT+B table is kidney inner medulla collecting duct and the one in the ontology is kidney inner medulla collecting duct principal cell. For reference, the given name/label by SMEs is Inner Medullary Collecting Duct Cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

3. Could not find any extra spaces. The term CL:1000454 has a different name/label in the source ontology in the following 1 row 52. The name/label in the ASCT+B table is kidney collecting duct epithelial cell _ and the one in the ontology is _kidney collecting duct epithelial cell. For reference, the given name/label by SMEs is kidney collecting duct epithelial cell . Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

4. Fixed 2 of 4 instances to podocyte. The term CL:0000653 has a different name/label in the source ontology in the following 2 rows 14, 20. The name/label in the ASCT+B table is glomerular visceral epithelial cell and the one in the ontology is podocyte. For reference, the given name/label by SMEs is Podocyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

5. Updated 2 instances to kidney loop of Henle long descending thin limb outer medulla recently changed in uberon. The term UBERON:8600036 has a different name/label in the source ontology in the following 4 rows 17, 17, 32, 32. The name/label in the ASCT+B table is long descending limb of the loop of Henle in the outer medulla _ and the one in the ontology is _kidney loop of Henle long descending thin limb outer medulla. For reference, the given name/label by SMEs is descending thin limb of loop of Henle 2. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

6. updated 1 instance kidney loop of Henle long descending thin limb inner medulla recently changed in uberon.The term UBERON:8600037 has a different name/label in the source ontology in the following 2 rows 33, 33. The name/label in the ASCT+B table is long descending limb of the loop of Henle in the inner medulla and the one in the ontology is kidney loop of Henle long descending thin limb inner medulla. For reference, the given name/label by SMEs is descending thin limb of loop of Henle 3. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

7. updated 1 instance ascending vasa recta, recently changed in uberon. The term UBERON:0009091 has a different name/label in the source ontology in the following 2 rows 64, 64. The name/label in the ASCT+B table is vasa recta ascending limb and the one in the ontology is ascending vasa recta. For reference, the given name/label by SMEs is Ascending Vasa Recta. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Kidney_v1.6_DRAFT
https://docs.google.com/spreadsheets/d/1CS-Tplkosorg8q6Ci8oALTvt9ufvEyDmCNAI2LsKM_g/edit?gid=949267305#gid=949267305

**Different labels**
1. Fixed last instance of space after label. The term CL:1000454 has a different name/label in the source ontology in the following 1 row 52. The name/label in the ASCT+B table is kidney collecting duct epithelial cell _ and the one in the ontology is _kidney collecting duct epithelial cell. For reference, the given name/label by SMEs is kidney collecting duct epithelial cell . Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
